client/packages/system/src/Item/api/hooks/useItemVariants/useUnitVariantList.ts

```diff
@@ -4,7 +4,5 @@ import { useItemApi } from '../useItemApi';
 export const useUnitVariantList = () => {
   const api = useItemApi();
 
-  return useQuery(api.keys.unitVariantsList(), () =>
-    api.get.unitVariantsList()
-  );
+  return useQuery(api.keys.packUnits(), () => api.get.packUnits());
```

Comment:
Is there a reason that the folder is `useItemVariants` and the file is `useUnitVariantList`? 🤔

Reply:
There is an issue for renaming... I jumped the gun and did it in places that I passed through 🙇🏻